README.md

```diff
@@ -49,12 +49,21 @@ In order to start contributing, simply follow these 10 steps:
 2. Fork the [ACS UPB Mobile](https://github.com/acs-upb-mobile/acs-upb-mobile) repository.
 3. Clone the fork onto your system and open it as an Android Studio project.
 4. Don't forget to run `flutter pub get` to update the dependencies.
-5. Add yourself to the list of contributors in the `README.md` file. The recommended format is: `[FirstName LastName](link_to_github_profile)`.
-6. Commit and push the changes to the forked repository (*Ctrl*+*K* for committing, *Ctrl*+*Shift*+*K* for pushing in Android Studio).
-7. Create a pull request either from GitHub (you will see a message prompting you to create a PR when you open the forked repository) or Android Studio (VCS > Git > Create Pull Request).
-8. Wait for the PR to be approved.
-9. ???
-10. Profit!
+5. Create a new branch
+6. Add yourself to the list of contributors in the `README.md` file. The recommended format is: `[FirstName LastName](link_to_github_profile)`.
+7. Commit and push the changes to the forked repository (*Ctrl*+*K* for committing, *Ctrl*+*Shift*+*K* for pushing in Android Studio).
+8. Create a pull request either from GitHub (you will see a message prompting you to create a PR when you open the forked repository) or Android Studio (VCS > Git > Create Pull Request). Make sure the PR points from the branch you created to the master branch in the acs-upb-mobile repo.
+9. Wait for the PR to be approved.
+10. ???
+11. Profit!
+
+Once in a while you should sync your fork's master with the upstream (the original repo) master. To do this:
+1. In the repo on your machine add a new remote that points to `acs-upb-mobile/master`, call it `upstream_master`
+2. Checkout `upstream_master` on a branch with the same name
+3. Pull
+4. Checkout your fork's master branch, should be called `master`
+5. Merge `upstream_master` into `master`. This may produce conflicts, tough luck. [KDiff3](http://kdiff3.sourceforge.net/) is a good tool to see the conflicts and solve them (it solves some automagically).
```

Comment:
```suggestion
4. Merge the remote `master` into your `master`: `git merge upstream/master`. This may produce conflicts, unfortunately. To fix them, open the *VCS* menu in Android Studio, select *Git* and *Resolve conflicts*.
```